src/recipes/ai-context/context.ts

```diff
@@ -137,3 +208,12 @@ export const writeFile = async (path: string, contents: string) => {
   await fs.mkdir(directory, { recursive: true })
   await fs.writeFile(path, contents)
 }
+
+export const deleteFile = async (path: string) => {
+  try {
+    // delete file from file system - not just unlinking it
+    await fs.rm(path)
+  } catch (_) {
```

Comment:
nit: No use for variable.

```suggestion
  } catch {
```